workflows/files/ZWEWRF06.xml

```diff
@@ -0,0 +1,251 @@
+        <category>Parmlib</category>
+        <string valueMustBeChoice="false" multiLine="false">
+            <validationType>DSNAME</validationType>
+        </string>
+    </variable>
+    <variable name="sched" scope="instance" visibility="public">
+        <label>Scheduler member</label>
+        <abstract>Please specify the name of the active SCHEDxx member</abstract>
+        <description>Please specify the scheduler member that is currently used in the specified PARMLIB.</description>
+        <category>Parmlib</category>
+        <string valueMustBeChoice="false" multiLine="false">
+            <validationType>DSMEMBERNAME</validationType>
+            <default>SCHED00</default>
+        </string>
+    </variable>
+    <!--atCreate-->
+    <step name="define_variables" optional="false">
+        <title>Define Variables</title>
+        <description>Define the variable values for the execution</description>
+        <!-- pre-requisite step -->
+        <variableValue name="zowe_install_path" scope="instance" noPromptIfSet="false" required="true"/>
+        <variableValue name="zowe_dsn_prefix" scope="instance" noPromptIfSet="false" required="true"/>
+        <variableValue name="xmem_prefix" scope="instance" noPromptIfSet="false" required="true"/>
+        <variableValue name="xmem_dasd" scope="instance" noPromptIfSet="false" required="true"/>
+        <variableValue name="proclib" scope="instance" noPromptIfSet="false" required="true"/>
+        <variableValue name="parmlib" scope="instance" noPromptIfSet="false" required="true"/>
+        <variableValue name="sched" scope="instance" noPromptIfSet="false" required="true"/>
+        <!-- condition -->
+        <instructions substitution="false">Run this step to define the variables for the workflow execution.</instructions>
+        <weight>1</weight>
+        <skills>None</skills>
+        <autoEnable>true</autoEnable>
+        <canMarkAsFailed>false</canMarkAsFailed>
+        <!--template-->
+    </step>
+    <step name="allocate_xmem" optional="false">
+        <title>Allocate Cross Memory Server Data Sets</title>
+        <description>Allocates the data sets for the Cross Memory Server.</description>
+        <prereqStep name="define_variables"/>
+        <!--variableValues-->
+        <!-- condition -->
+        <instructions substitution="false">Use this step to allocate the data sets required for XMEM</instructions>
+        <weight>1</weight>
+        <skills>z/OS System Programmer</skills>
+        <autoEnable>true</autoEnable>
+        <canMarkAsFailed>false</canMarkAsFailed>
+        <template>
+            <inlineTemplate substitution="true">//ALLOCDSS EXEC PGM=IEFBR14
+//LOADLIB1 DD DISP=(NEW,CATLG,DELETE),
+//    DSN=${instance-xmem_prefix}.LOADLIB,
+//    DSNTYPE=LIBRARY,SPACE=(CYL,(10,5,5)),
+//    DCB=(RECFM=U),
+//    ${instance-xmem_dasd}
+//PARMLIB1 DD DISP=(NEW,CATLG,DELETE),
```

Comment:
use DISP=(**MOD**,CATLG,DELETE), to support the customer specifying an existing data set